aws/scripts/py/cache_invalidation.py

```
@@ -0,0 +1,97 @@
+import json
+import subprocess
+import os
+import time
+
+def get_cloudfront_distributions():
```

Ruff (0.11.9)

Missing return type annotation for public function `get_cloudfront_distributions`

(ANN201)